TobiiTest/OCRUtil.cs

```diff
@@ -26,7 +26,27 @@ class OCRUtil
             { "eng", "English" },
             { "fin", "Finnish" },
             { "fra", "French" },
-            { "rus", "Russian" }
+            { "rus", "Russian" },
```

Comment:
Can you sort languages alphabetically? So it looks prettier in the UI

Reply:
ok